src/lib/prepareOas.ts

```diff
@@ -65,9 +65,18 @@ export default async function prepareOas(path: string, command: 'openapi' | 'ope
           const oas = new OASNormalize(file, { enablePaths: true });
           return oas
             .version()
-            .then(version => {
-              debug(`OpenAPI/Swagger version for ${file}: ${version}`);
-              return version ? file : '';
+            .then(({ specification, version }) => {
+              if (specification === 'openapi') {
+                debug(`OpenAPI version for ${file}: ${version}`);
+              } else if (specification === 'swagger') {
+                debug(`Swagger version for ${file}: ${version}`);
+              } else if (specification === 'postman') {
+                debug(`Postman version for ${file}: ${version}`);
```

Comment:
`version` here will be "unknown" right now because we don't have any way in `oas-normalize` at the moment to know if a collection is v2.0 or v2.1. Maybe this debug message should be something different?

Reply:
I know it's a bit hairy to look at the `info.schema` field, but I'd love if we could parse the Postman version out of there (and also use that to determine if it is a postman collection or not). I have yet to see a schema field that isn't `https://schema.getpostman.com/json/collection/v2.1.0/collection.json` or `https://schema.getpostman.com/json/collection/v2.0.0/collection.json`.